build.gradle.kts

```diff
@@ -17,19 +17,95 @@ plugins {
     `kotlin-dsl`
     id("maven-publish")
     id("signing")
-    id("org.nosphere.apache.rat") version "0.6.0"
+    id("org.jetbrains.kotlin.jvm")
+    id("java-gradle-plugin")
+    id("com.gradle.plugin-publish")
+    id("org.nosphere.apache.rat")
+    `knotx-plugins-release-base-tmp`
 }
 
+repositories {
+    jcenter()
+    gradlePluginPortal()
+}
+
+dependencies {
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.3.61"))
+
+    implementation(gradleApi())
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

Comment:
Does it mean that when migrating to jdk8+ we would have to update this dependency?